style.css

```diff
@@ -408,3 +433,262 @@ input {
   margin-top: 32px;
   margin-bottom: 8px;
 }
+
+.desktopWorks {
+  display: none;
+}
+
+#getResume {
+  size: 20px;
+  line-height: 24px;
+  font-size: 17px;
+  color: #396df2;
+  border-radius: 8px;
+  border: 1px solid #396df2;
+  background-color: white;
+  margin-top: 5%;
+  margin-bottom: 2%;
+}
+
+@media (min-width: 768px) {
+  .backgroundImg {
+    background-image: url(img/Header\ bg@2x.png);
+    background-repeat: no-repeat;
+    background-size: 100% 120%;
+    margin-bottom: 4rem;
+  }
+
+  .menu li {
+    padding-right: 3rem;
+    color: #344563;
+  }
+
+  .menu ul {
+    display: flex;
+    justify-content: flex-end;
+    list-style: none;
+    gap: 20px;
+    color: #344563;
+  }
+
+  .menu > div {
+    display: none;
+  }
+
+  .Personal_info {
+    padding-left: 20%;
+    align-self: center;
+    padding-top: 15%;
+  }
```

Comment:
- [ ] Regarding the figma design let adjust the the margin on this section to make the match, I suggest to justify this section center or use margins:
- Your design:
![topyYours](https://user-images.githubusercontent.com/97834160/206593950-aba30d5c-01e7-4ff9-8141-ad709bf75927.png)


- Figma design:
![topFig](https://user-images.githubusercontent.com/97834160/206593975-6c298916-836f-4acd-ac7a-2ee346028401.png)